docs/technical/target_types.md

```diff
@@ -34,24 +34,7 @@ spec:
 
 **Config Variant 2**:
 
-This variant contains the kubeconfig in a secrets referenced in the `config` section. The key in the data section, where
-to find the kubeconfig, could be specified. Currently, the secret must be in the same namespace as the target.
-
-```yaml
-apiVersion: landscaper.gardener.cloud/v1alpha1
-kind: Target
-metadata:
-    name: ...
-    namespace: ...
-spec:
-    config:
-      secretRef:
-        name: my-secret
-        namespace: default
-        key: kubeconfig # optional will default to "kubeconfig"
-```
-
-**Config Variant 3**:
+**DEPRECATED: use the Target's `spec.secretRef` field instead**
```

Comment:
Should we perhaps add a sentence like? The variant with a reference in the kubeconfig section if deprecated...

Reply:
Where do you want to add this sentence?

Reply:
Perhaps below the deprecation hint?

Reply:
I'd say we don't need another sentence, as the current hint seems pretty obvious to me, but I don't have a problem with adding one either. I'll add one.

Reply:
I only thought, if a user sees a target with the old format, how should he know that this is variant 3, if this format is not mentioned anymore?

Reply:
But if you do not agree, I'm totally fine with the current version.